android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/dao/UploadDAO.java

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.bridge.android.manager.dao;
+
+import android.content.Context;
+
+import com.google.common.collect.Sets;
+
+import org.sagebionetworks.bridge.android.manager.UploadManager;
+import org.sagebionetworks.bridge.rest.model.UploadSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Set;
+
+/**
+ * Created by jyliu on 3/10/2017.
+ */
+
+public class UploadDAO extends SharedPreferencesJsonDAO {
+    private static final Logger logger = LoggerFactory.getLogger(UploadDAO.class);
+
+    private static final String PREFERENCES_FILE  = "uploads";
```

Comment:
The more I think about it, we should move all the DAOs form shared preferences to StorageAccess.  That way we can just delete all the files in StorageAccess when the user signs out and it will make life a lot easier.